test/lib/ls-utils.test.ts

```diff
@@ -215,14 +228,3 @@ test("parse ls output", async (t) => {
     });
   }
 });
-
-const getLSOutputFixture = (file: string) =>
```

Comment:
This was some testing code which I mistakenly checked in some time ago ... Removing this should speed up the tests :smile:

Reply:
would be great to have this kind of things in a separate commit, at least for next time :) thanks for the heads up!